client/src/js/services/color.js

```diff
@@ -0,0 +1,22 @@
+angular.module('bhima.services').service('ColorService', ColorService);
```

Comment:
Nice!

Can you write a quick unit test to make sure that `service.list` returns an array?